PRASFiles.jl/test/runtests.jl

```diff
@@ -20,11 +20,21 @@ using JSON3
         rts2 = SystemModel(path * "/rts2.pras")
         @test rts == rts2
 
-        savemodel(rts,path * "rts_userattrs.pras",
-        user_attributes=Dict("about"=>"this is a representation of the RTS GMLC system"))
-        user_attrs = PRASFiles.read_addl_attrs(path * "rts_userattrs.pras") 
+        savemodel(rts,path * "/rts_userattrs.pras",
+            user_attributes=Dict("about"=>"this is a representation of the RTS GMLC system"))
+            
+        user_attrs = PRASFiles.read_addl_attrs(path * "/rts_userattrs.pras") 
         @test user_attrs == Dict("about"=>"this is a representation of the RTS GMLC system")
 
+        rts_userattrs = SystemModel(path * "/rts_userattrs.pras")
+        @test_throws "Attribute 'about' already exists in the system" savemodel(rts_userattrs,path * "/rts_userattrs.pras",
+                                                                                user_attributes=user_attrs)
+
+        savemodel(rts_userattrs, path * "/rts_userattrs2.pras")
+        rts_userattrs2 = SystemModel(path * "/rts_userattrs2.pras")
+        @test rts_userattrs == rts_userattrs2
+
+
```

Comment:
Can we add a test which tests full round-trip of read and write of a SystemModel with attributes?

Reply:
Do we want to include tests for printing like these (https://github.com/NREL-Sienna/InfrastructureSystems.jl/blob/main/test/test_printing.jl) or the ones in PSY?